README.md

```diff
@@ -89,6 +89,7 @@ const format = require('format-package');
 const pkg = require('<path-to-package.json>');
 
 async function formatPackage(pkg) {
+  const options = {parser: 'json'}
```

Comment:
the options passed in to the format method in this case refers to the options defined on [line 107](https://github.com/camacho/format-package/pull/23/files#diff-04c6e90faac2675aa89e2176d2eec7d8R107). 

If I understand correctly, `{parser: 'json'}` needs to be passed in to the `prettier.format` method. If that is the case, I believe the change belongs on [line 312](https://github.com/camacho/format-package/pull/23/files#diff-04c6e90faac2675aa89e2176d2eec7d8R312):

```js
const formatPkg = require('format-package');
const prettier = require('prettier');
const pkg = require('./package.json');
 	 
const formatter = async content => {
    const options = await prettier.resolveConfig('./package.json');
    // include `{ parser: 'json' }` in the options passed to prettier 
    return prettier.format(content, {...options, parser: 'json'});
};
 	 
formatPkg(pkg, { formatter });
```